src/modules/data.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
- [ ] For best practice reasons and since it is a common practice in real working environment, I suggest fixing this listing issue instead of disabling linter checks here. Kindly find a way to resolve this without obstructing linter checks :pray: 

- Consider renaming your files without any underscore or hyphens. Camel casing would be preferable :asterisk: